inst/rmd/qc_report.rmd

```diff
@@ -73,10 +82,21 @@ unfiltered_meta <- metadata(unfiltered_sce)
 sample_information <- tibble::tibble(
   "Sample id" = sample_id,
   "Tech version" = format(unfiltered_meta$tech_version), # format to keep nulls
+  "Data modalities" = paste(modalities, collapse = ", "),
   "Number of reads sequenced" = format(unfiltered_meta$total_reads, big.mark = ',', scientific = FALSE),
   "Percent of reads mapped" = paste0(round((unfiltered_meta$mapped_reads/unfiltered_meta$total_reads) *100, 2), "%"),
   "Number of cells reported by alevin-fry" = format(unfiltered_meta$af_num_cells, big.mark = ',', scientific = FALSE)
-) %>%
+)
+if (has_cite){
+  cite_meta <- metadata(altExp(unfiltered_sce, "CITEseq"))
+  sample_information <- sample_information %>%
+    mutate(
+      "CITE-seq reads sequenced" = format(cite_meta$total_reads, big.mark = ',', scientific = FALSE),
+      "CITE-seq reads mapped" = format(cite_meta$mapped_reads, big.mark = ',', scientific = FALSE),
```

Comment:
I think this should also be percent reads for CITE-seq to be consistent with the RNA-seq reporting above?

Reply:
Yup! I thought I had done this, but apparently not!

Reply:
Oh, I see the issue... I had actually meant to delete these lines, as they are redundant with the info in the CITE-seq section (where I do have it as % mapped%). Do you think we should have this info in both places, or should I delete it?